### PR TITLE
New Updates

### PR DESCRIPTION
**Pull request by Sonichigo**
## Description

- [x] I have followed the [code of conduct](https://github.com/Alphasians/AINA/blob/main/contributing.md)
- [x] I have followed the [contributing guidelines](https://github.com/Alphasians/AINA/blob/main/contributing.md).
- [x] Either I have tested the code or my changes are not related to code.

## Related Issue

fixes #

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,13 +13,14 @@
     "Krishdevdb"
```
Comment (krishguptadev) [inline package.json:13]: hmm?

Comment (krishguptadev) [reply package.json:13]: Do you really wanna maintain this list?

File: package.json [hunk 2]
```diff
@@ -13,13 +13,14 @@
...
   "directories": {
-    "src": "src",
-    "lib": "dist"
+    "doc": "docs"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Alphasians/AinaBot.git"
   },
-  "repository": "github:Alphasians/AinaBot",
```
Comment (krishguptadev) [inline package.json:22]: This is the correct way to add a github repo.

File: package.json [hunk 3]
```diff
@@ -35,6 +36,7 @@
     "discord.js-commando": "github:discordjs/Commando",
     "dotenv": "^10.0.0",
     "ffmpeg-static": "^4.4.0",
+    "nodemon": "^2.0.15",
```
Comment (krishguptadev) [inline package.json:39]: should be dev-dep

File: src/commands/misc/profile.js [hunk 4]
```diff
@@ -1,14 +1,13 @@
...
+const { MessageEmbed } = require('discord.js')
+const Levels = require('discord-xp')
 
 Levels.setURL(process.env.mongoPath)
 
 module.exports = class Level extends Command {
   constructor (client) {
     super(client, {
       name: 'profile',
-      aliases: [],
```
Comment (krishguptadev) [inline src/commands/misc/profile.js:11]: I see a possible alias of `!me` here

File: package.json (hunk 1 above)
Comment (Sonichigo) [reply package.json:13]: Yep

Comment (krishguptadev) [reply package.json:13]: Then you need to add a lot of other ppl as well 😅

**Pull request merged**